### PR TITLE
Show empty list when search text is empty

### PR DESCRIPTION
**Pull request by yatsukhnenko**
To not execute fetch when search test is empty I used `execute` option of `useFetch` hook.
Also I used `List.EmptyView` when search text is empty.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -6,15 +6,9 @@ import { ActionPanel, Action, List, Icon } from "@raycast/api";
...
-    // Don't fire search if query is empty
-    // https://github.com/rhubarbgroup/raycast-wordpress-docs/issues/1
-  }
-
-  const query = searchText.length === 0 ? "get_" : searchText;
-
-  const { data, isLoading } = useFetch(`https://developer.wordpress.org/search/${query}/feed/rss/`, {
+  const { data, isLoading } = useFetch(`https://developer.wordpress.org/search/${searchText}/feed/rss/`, {
     parseResponse: parseFetchResponse,
+    execute: searchText.length > 0,
```
Comment (tillkruss) [inline src/index.tsx:11]: Great find!

File: src/index.tsx [hunk 2]
```diff
@@ -24,11 +18,15 @@ export default function Command() {
...
-      </List.Section>
+      {searchText.length > 0 ? (
+        <List.Section title="Results" subtitle={data?.length + ""}>
+          {data?.map((searchResult: SearchResult) => (
+            <SearchListItem key={searchResult.name} searchResult={searchResult} />
+          ))}
+        </List.Section>
+      ) : (
+        <List.EmptyView title={"Type to search"} icon={{ source: "command-icon.png" }} />
+      )}
```
Comment (tillkruss) [inline src/index.tsx]: Can you a screenshot of how it looks when no query is typed, and when no results where found?

**Review by tillkruss — CHANGES_REQUESTED** [review]

File: src/index.tsx (hunk 2 above)
Comment (yatsukhnenko) [reply src/index.tsx]: https://user-images.githubusercontent.com/1374472/231369986-ddf3e095-f6ac-46cb-a217-7ca7e2408166.mov

Comment (yatsukhnenko) [reply src/index.tsx]: @tillkruss Here is short screen recording to show how it works

Comment (yatsukhnenko) [reply src/index.tsx]: Do I also need using `List.EmptyView` when search query returns no results?

Comment (tillkruss) [reply src/index.tsx]: > Do I also need using `List.EmptyView` when search query returns no results?

No, but let's remove the WordPress icon before a query was typed, it's too big.

Comment (yatsukhnenko) [reply src/index.tsx]: Done. Now it looks exactly the same as when no result. Do you want me to create another video?

Comment (tillkruss) [reply src/index.tsx]: Thanks for asking. No this looks fantastic!

**Pull request merged**